aten/src/ATen/templates/Functions.cpp

```diff
@@ -137,7 +142,7 @@ inline DataPtr TensorMaker::makeDataPtrFromContext() noexcept {
 }
 
 IntArrayRef TensorMaker::makeTempSizes() const noexcept {
-  static int64_t zeros[5] = {0, 0, 0, 0, 0};
+  static std::int64_t zeros[5] = {0, 0, 0, 0, 0};
```

Comment:
the need to `std::` here is weird

Reply:
Well I am slightly obsessed in this regard. For C standard library typedefs and functions declared/defined in `<c*>` headers I always use namespaced forms. Similarly when using C APIs I always scope them with `::<name>`. I know that is more pedantic and verbose, but I just don't like mixing up language keywords and C-style type/function names; especially typedefs for primitive types such as the ones in `<cstddef>`.